pages/dashboard/AllEntries/index.js

```diff
@@ -41,41 +43,93 @@ const StyledBatch = styled.div`
   border-bottom: 2px solid #8c907a;
 `;
 
+const StyledFilter = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const StyledSelect = styled.select`
+  border-radius: 22px;
+  border: 1px solid #ececea;
+  padding: 2vw;
+  background-color: #d7b470;
+  width: 35vw;
+`;
+const listOfTagOptions = [
+  { tagName: "Family" },
+  { tagName: "Friends" },
+  { tagName: "Partner" },
+  { tagName: "Work" },
+  { tagName: "Hobby" },
+  { tagName: "Household" },
+  { tagName: "TV" },
+  { tagName: "Sports" },
+  { tagName: "Walk" },
+];
```

Comment:
It could be enough to create an array with strings because your key prop is always the same.

```suggestion
const listOfTagOptions = ["Family", "Friends", "Partner", "Work", "Hobby", "Household", "TV", "Sports", "Walk"];
```

Reply:
Thank you @stefankorder ! :) 
Yeah you're absolutely right, I just kept it like that because I use this list in different parts of the App where I do need the keys for the logic & want to refactor it by uplifting it later on so it has to stay like that. :) 

So may I'm allowed to go on with this version anyway ? :)

Reply:
okay :)